Assets/Spawner/Spawner.cs

```diff
@@ -659,18 +818,25 @@ public override IList<Wave> GetChildren() {
     public override bool AddWave(int index, Wave wave) {
       return false;
     }
+
+    public override void GetSpawnTimes(ref EnemySpawnTimes spawnTimes, ref float projectedStartTime) {
+      if (Finished) return;
```

Comment:
Dialogue boxes are another indeterminate wave that should short circuit the spawn time method.
Things are a little complicated here, i would set start time to when the dialogue box is closed and then do as with spacer waves to calculate how much time is left.

Reply:
Done.